src/amp-ads/40_Experimental_Ads/Scrollbound_Video_Ad.html

```diff
@@ -0,0 +1,263 @@
+<!---{
+ "preview": "a4a-experimental",
+ "skipValidation": true,
+ "hidePreview": true,
+ "experiments": ["amp-animation"],
+ "width": 350,
+ "height": 250,
+ "layout": "responsive",
+ "scrollToReveal": true
+}--->
+<!-- -->
+<!doctype html>
+<html ⚡4ads>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+  <!-- TODO(aghassemi, #9311): Put real binaries when #9311 is resolved -->
```

Comment:
Can you remove this comment as it will show up in the docs. ;-) Or rephrase it to something like:

```
  <!-- ## Setup -->
  <!-- 
  The features used in this sample are still experimental and require a custom version of the AMP runtime.
  -->
```